docs/helm-chart/manage-dags-files.rst

```diff
@@ -144,6 +144,11 @@ hence Webserver does not need access to DAG files, so ``git-sync`` sidecar is no
 Notes for combining git-sync and persistence
 ............................................
 
+.. note::
+
+    When using the helm chart, it is not possible to use git-sync and persistence at the same time
```

Comment:
```suggestion
    When using the helm chart, it is not recommended to use git-sync and persistence at the same time
```

Reply:
I believe @mashanm mentioned it is technically not even possible to use both at the same time when using the Helm Chart.

And this is undocumented.

Reply:
if it's not possible we should change this line too.
`While using both git-sync and persistence for DAGs is possible`